compose/django.md

```diff
@@ -138,21 +146,24 @@ In this section, you set up the database connection for Django.
 
 1.  In your project directory, edit the `composeexample/settings.py` file.
 
-2.  Replace the `DATABASES = ...` with the following:
-
-        DATABASES = {
-            'default': {
-                'ENGINE': 'django.db.backends.postgresql',
-                'NAME': 'postgres',
-                'USER': 'postgres',
-                'HOST': 'db',
-                'PORT': 5432,
-            }
-        }
-
-    These settings are determined by the
-    [postgres](https://hub.docker.com/_/postgres) Docker image
-    specified in `docker-compose.yml`.
+2. Replace the `DATABASES = ...` with the following:
+
+       # setting.py
+       
+       DATABASES = {
+           'default': {
+               'ENGINE': 'django.db.backends.postgresql',
+               'NAME': 'postgres',
+               'USER': 'postgres',
+               'PASSWORD': 'postgres',  #add line.
```

Comment:
Please, remove the comment "#add line" from it

Reply:
Roger